packages/vc-verification/src/issuer-verification.ts

```diff
@@ -0,0 +1,255 @@
+// @ts-ignore
+import * as jwt from 'jsonwebtoken';
+import { utils, providers } from 'ethers';
+import { ProofVerifier } from '@ew-did-registry/claims';
+import { Resolver } from '@ew-did-registry/did-ethr-resolver';
+import { RegistrySettings } from '@ew-did-registry/did-resolver-interface';
+import { RoleDefinitionResolverV2__factory } from '@energyweb/credential-governance/ethers/factories/RoleDefinitionResolverV2__factory';
+import { RoleDefinitionResolverV2 } from '@energyweb/credential-governance/ethers/RoleDefinitionResolverV2';
+import { StorageSettings } from './models';
+import { CredentialResolver } from '.';
+import { IVerifiableCredential, VerificationResult } from './models';
+
+export class IssuanceVerification {
+  private _provider: providers.Provider;
+  private _resolver: Resolver;
+  private _registrySetting: RegistrySettings;
+  private _roleDefResolver: RoleDefinitionResolverV2;
+  private _credentialResolver: CredentialResolver;
+
+  /**
+   *
+   * @param provider
+   * @param roleDefResolverAddr
+   * @param registrySetting
+   * @param storageSettings
+   */
+  constructor({
+    provider,
+    roleDefResolverAddr,
+    registrySetting,
+    storageSettings,
+  }: {
+    provider: providers.Provider;
+    roleDefResolverAddr: string;
+    registrySetting: RegistrySettings;
+    storageSettings: StorageSettings;
```

Comment:
Maybe consider using name of `credentialResolverSettings` to match the `CredentialResolver` class where this param is used.
```suggestion
    storageSettings: StorageSettings;
```